src/tools/style-builder-tool/StyleBuilderTool.schema.ts

```diff
@@ -99,18 +127,18 @@ export const StyleBuilderToolSchema = z.object({
   base_style: z
     .enum([
       'standard',
-      'streets',
-      'light',
-      'dark',
-      'satellite',
-      'outdoors',
-      'blank'
+      'streets-v11',
+      'light-v10',
+      'dark-v10',
+      'satellite-v9',
+      'satellite-streets-v11',
+      'outdoors-v11'
```

Comment:
We have a newer version of those styles.
https://docs.mapbox.com/api/maps/styles/#classic-mapbox-styles